p2p/router.go

```diff
@@ -149,6 +158,18 @@ func (o *RouterOptions) Validate() error {
 		return fmt.Errorf("queue type %q is not supported", o.QueueType)
 	}
 
+	switch {
+	case o.IncomingConnectionWindow == 0:
+		o.IncomingConnectionWindow = 100 * time.Millisecond
```

Comment:
Is 100ms kind of arbitrary or do you have some gut judgement here? (totally cool with either answer :-p)

Reply:
some of both? Mostly arbitrary, but also I can't imagine connection attempts more frequent than 100ms being a viable use case?